src/helpers.ts

```diff
@@ -66,7 +66,7 @@ export function saveDistributeEvent(
   let distEvents = tx.distributionEvents;
   if (!distEvents) distEvents = new Array<string>();
 
-  let distEventId = createJointId([txHash, logIdx.toString()]);
+  let distEventId = createJointId([DISTRIBUTION_EVENT_PREFIX, txHash, logIdx.toString()]);
```

Comment:
@wminshew any issue with the changes in this commit? I wasn't sure if these were just oversights or left out for a reason.